tools/lkl/cptofs.c

```diff
@@ -1,3 +1,5 @@
+#define _GNU_SOURCE
```

Comment:
Not sure if this is a problem? What operating systems do we need to support here?